v2/dtos/responses/device.go

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package responses
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common"
+)
+
+// AddDeviceResponse defines the Response Content for POST Device DTOs.
+// This object and its properties correspond to the AddDeviceResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/AddDeviceResponse
+type AddDeviceResponse struct {
+	common.BaseResponse `json:",inline"`
+	Id                  string `json:"id"`
+}
+
+// UpdateDeviceResponse defines the Response Content for PUT Device as pushed DTO.
+// This object and its properties correspond to the UpdateDeviceResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/UpdateDeviceResponse
+type UpdateDeviceResponse struct {
```

Comment:
I see that we'll have many Response DTOs that are identical. Can we have a more generic `AddUpdateResponse` that is used instead of all these identical Response DTOs? @cloudxxx8 , @iain-anderson , @tonyespy ??

Reply:
Still need to address this comment, simply need a common response DTO that includes the Id property. `BaseWithIdresponse` or `SimpleIdResponse` or `IdResponse` ???